.travis/script.sh

```diff
@@ -68,3 +68,13 @@ make all
 end_section "symbiflow.build_all_demos"
 
 $SPACER
+
+start_section "symbiflow.run_python_tests" "Run Python unit tests"
+# Reset working directory
+cd /home/travis/build/SymbiFlow/symbiflow-arch-defs
+pip install -r ./requirements.txt
+python -m unittest utils.lib.rr_graph.tests.test_channel
+python -m unittest utils.lib.rr_graph.tests.test_graph
```

Comment:
I think it'd be good to run tests via a build system target. We want to make sure we are testing against the python and libs from conda. I see we already have a [pytest](https://github.com/SymbiFlow/symbiflow-arch-defs/blob/4101e067a4174e5bb7f54e0cb3f9bad222c14e0e/utils/CMakeLists.txt) target

I realize the dependencies aren't clean, so created #463 to improve the situation with python dependencies.

Reply:
@Fleker Just a heads up, I probably just caused a conflict by moving this block up in #480